### ISSUE TITLE
Desktop Safari 11 autoplay w/o muted never plays ad or content

### ISSUE THREAD

**Issue by marguinbc** [googleads/videojs-ima#435]
The latest IMA SDK 3.182.0, has changed the autoplay behavior for click-to-play ads but has seemed to negatively impact autoplay ads so that when failing to autoplay, they cannot be played at all.

If you modify the autoplay sample by removing the` muted` attribute and load the page on Safari 11 Desktop you will see a Unhandled Promise Rejection in the console and the player will display the big-play-button and poster:

https://googleads.github.io/videojs-ima/examples/autoplay/

However, at that point, clicking the play button does not start the ad or content and the player is in a stuck/frozen state.

It seems like this would be an IMA-SDK issue since I think it would be handling the promise and not the videojs-ima plugin or videojs in this instance but wanted to log it here as well.

**Comment by shawnbuso** [comment]
Safari 11 no longer supports unmuted autoplay. We're working on a change to fall back to click to play when autoplay fails, but it's currently blocked by an open issue with the IMA SDK.

**Comment by marguinbc** [comment]
Hi @shawnbuso, fair enough. I thought the issue was in the IMA SDK as we are seeing the same behavior in our implementation.

**Comment by marguinbc** [comment]
@shawnbuso, just so I understand your last comment, understanding that Safari 11 Desktop does not autoplay unless the video is muted.  

Is there a remaining issue in the IMA SDK where it is not failing gracefully when the autoplay attribute is present and the play() promise is rejected?  

And is this different than the documented fix for "click-to-play" in Safari 11 released in 3.182.0?

**Comment by shawnbuso** [comment]
>Is there a remaining issue in the IMA SDK where it is not failing gracefully when the autoplay attribute is present and the play() promise is rejected?

Yes, that is correct. We're going to be adding a new error event to the SDK that will fire in this scenario, and the implementation can listen for that error and re-request ads to be played on user click. That work is still in progress.

**Comment by shawnbuso** [comment]
The IMA SDK issue mentioned above has been fixed, but we still don't have a good solution for autoplay with the plugin. I'm going to close this out since at this point it's just looking for a good solution for autoplay, which is a duplicate of #341. Thanks!
